lib/rules/indent.js

```diff
@@ -651,7 +651,7 @@ module.exports = {
             if (elements.length > 0 && node.type === "ArrayExpression") {
                 elements.forEach(el => {
                     if (el.loc.start.line === node.loc.start.line) {
-                        return;
+                        return; // eslint-disable-line no-useless-return
```

Comment:
I think we can just remove the entire `elements.forEach` block (see https://github.com/eslint/eslint/pull/7237#discussion_r85409147)